CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Update minimum ZAP version to 2.16.0 and compile with Java 17.
 - Add cautionary note to help and readme.
 - Maintenance and documentation changes.
+- Updated scan_rule_list.js to work with ZAP 2.16.0 and later.
```

Comment:
What's not working? This falls under the previous entry IMHO.

Reply:
My comment is misleading I guess. It isn't that it wasn't/didn't work. It's more that I figured it should now be leveraging the add-on functionality.

Ok, I'll remove the separate note.